.travis.yml

```diff
@@ -34,6 +34,42 @@ jobs:
           install:
               - travis_retry composer update --classmap-authoritative --no-suggest  --prefer-dist --prefer-lowest --prefer-stable
 
+        - stage: Test
+          env: DB=MariaDB
+          php: 7.2
+          addons:
+              mariadb: 10.0
+          before_script:
+              - mysql --execute='CREATE DATABASE test;'
+          script:
+              - DB_URI=mysql://travis@localhost/test vendor/bin/phpunit --group database
```

Comment:
I take that this is not run locally in favor of Sqlite. If we find it useful to run this variant locally we can go for Docker solutions.